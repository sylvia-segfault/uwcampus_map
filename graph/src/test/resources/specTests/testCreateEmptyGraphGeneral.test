# Create an empty graph
CreateGraph graph4