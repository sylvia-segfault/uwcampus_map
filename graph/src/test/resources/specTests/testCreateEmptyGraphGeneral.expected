# Create an empty graph
created graph graph4